governance/polls/Ratification Poll - Adding a SPF - Makershire Hathaway (MIP55c3-SP4) - June 13, 2022.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Ratification Poll for Adding a SPF - Makershire Hathaway (MIP55c3-SP4) - June 13, 2022
+summary: The Makershire Hathaway SPF will be a 10M pool managed by the SF CU under the supervision of Risk CU and prospective LOVE CU to experiment with various yield generation strategies and support DAI liquidity/utility.
```

Comment:
"The Makershire Hathaway SPF will be a 10M pool managed by the Strategic Finance CU (SF-001) under the supervision of the Risk CU (RISK-001) and the prospective Lending Oversight CU (LOVE-001) to experiment with various yield generation strategies and support DAI liquidity/utility"